---
layout: page
title: About
permalink: /about/
---

## Hyunwoo Cho

Email: hc27oclock at gmail dot com

Actively seeking entry-level Ph.D. jobs in the United States.

## Projects

### Effect and importance of FOXA2 in pancreatic development from pluripotent stem cells
Co-first author of the publication (aiming for Cell Reports upon review by Cell Stem Cell).
Managed the computational analysis of transcriptomes, chromatin accessibility, histone modifications, and transcription factor binding.
Identified the chromatin groups accessible in specific patterns in pancreatic development and/or susceptible by FOXA2 knockout using CRISPR/Cas9 technology.
Associated the accessible chromatin groups with transcription factors using motif analysis, and showed that the relevance of FOXA2 increases in later stage of development.
Showed that positioned nucleosomes with histone H3 K4 monomethylation are identified with chromatin groups that are located as primed enhancers.

### Transcription factor motif analysis of ATAC-seq loci groups identified by overexpression of mutant FOXA1
The work has been submitted to Nature (under minor revision).
I developed a general-purpose process to:
- define modules in chromatin accessibility profiles
- find preferentially enriched transcription factor motifs and visualize them in bar plots
- associate each of them with the change of gene expression in respective stages.
Then I applied it to visualize the response by different genotypes of FOXA1.

### Analysis to show Ikzf2 binding in MLL-AF9 leukemia stem cells
Managed the computational analysis of transcriptomes, chromatin accessibility, and Ikzf2 binding.
Defined the epigenetically relevant differentially accessible chromatin regions and showed highly enriched transcription factors in them using motif analysis.
Showed the evidence of Ikzf2 binding in differentially accessible chromatin regions by CUT&RUN sequencing.

### Genomic analysis of memory in regulatory T cells
Performed transcriptome analysis and preprocessed chromatin accessibility data. Specifically, I provided a rigorous definition of RNA-seq gene groups.
Performed the association of Foxp3-binding peaks with the level of histone H3 K27 trimethylation.
Also managed the computational analysis part during the peer review process.
 

